src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsDF.scala

```diff
@@ -128,7 +128,7 @@ object FireAlertsDF {
     val confCols = if (!aggCol.equals("burned_area__ha")) List($"confidence__cat")  else List()
 
     val fireCols = List(
-      year($"alert__date") as "alert__year",
+      expr("EXTRACT(YEAROFWEEK FROM alert__date)") as "alert__year",
```

Comment:
Looks good.

Totally ignorable suggestion:  is it worth making line 132 be more obviously fitting the semantics of line 131, by changing it to :

```
  expr("EXTRACT(WEEK FROM alert__date)") as "alert_week",
```

Reply:
good question, I will confirm later if that does the same thing, but might merge this to get it into the data update branch